.markdownlintrc

```diff
@@ -3,5 +3,6 @@
   "line_length": false,
   "MD024": { "allow_different_nesting": true },
   "MD026": { "punctuation": ".,;:!" },
-  "DM046": { "style": "fenced" }
+  "DM046": { "style": "fenced" },
+  "MD033": false
```

Comment:
Assuming we don't want to open the floodgate to all inline-html shenanigans, a more targeted configuration that I have verified works would be:


```suggestion
  "MD033": { "allowed_elements": [ "details", "summary" ] }
```

Reply:
thanks, updated in ea2060bb257637fe0648d0cf4e8f75bf7a520223